lib/spoom/cli/commands/coverage.rb

```diff
@@ -11,21 +11,21 @@ module Commands
       class Coverage < Thor
         include Spoom::Cli::CommandHelper
 
+        DATA_DIR = "spoom_data"
+
         default_task :snapshot
 
         desc "snapshot", "run srb tc and display metrics"
-        option :save, type: :boolean, desc: "Save snapshot data as json"
-        option :save_dir, type: :string, desc: "Save json data under a specific directory", default: "spoom_data"
+        option :save, type: :string, desc: "Save snapshot data as json", lazy_default: DATA_DIR
         def snapshot
           in_sorbet_project!
 
           path = exec_path
           snapshot = Spoom::Coverage.snapshot(path: path)
           snapshot.print
 
-          save_dir = options[:save] ? options[:save_dir] : nil
+          save_dir = options[:save]
           return unless save_dir
```

Comment:
I don't think we need this anymore. I don't know if you want to still keep it.

Reply:
It's not here anymore: 
![image](https://user-images.githubusercontent.com/583144/95521755-51fac500-0998-11eb-87e9-aa702074b4a3.png)

Reply:
Wait I'm confused now 😄 

I meant I don't think we need `return unless save_dir`. Since `save_dir` will always exist due to the default value right? I assume that's how `lazy_default` works.

Reply:
With `lazy_default`, `options[:save]` returns:
* `X` if `--save X` was passed
* the lazy default value if `--save` was passed
* and `nil` otherwise